gnovm/tests/imports.go

```diff
@@ -254,6 +254,16 @@ func TestStore(rootDir, filesPath string, stdin io.Reader, stdout, stderr io.Wri
 				pkg.DefineGoNativeValue("Pi", math.Pi)
 				pkg.DefineGoNativeValue("MaxFloat32", math.MaxFloat32)
 				pkg.DefineGoNativeValue("MaxFloat64", math.MaxFloat64)
+				pkg.DefineGoNativeValue("MaxUint32", math.MaxUint32)
+				pkg.DefineGoNativeValue("MaxUint64", math.MaxUint64)
```

Comment:
I forgot to explicitly define the type here. 
```suggestion
				pkg.DefineGoNativeValue("MaxUint64", uint64(math.MaxUint64))
```